Server/routes/elementRoutes.js

```diff
@@ -32,6 +29,7 @@ router.post('/add', elementController.addElement);
 
 router
     .route('/:id')
+    .get(elementController.getElement)
```

Comment:
You don't want to allow non-admin users to get an Element ?

Reply:
Admin check is after